app/helpers/user.js

```diff
@@ -1,9 +1,20 @@
 const { obtainOneUser } = require('../services/users');
 const { verifyToken } = require('../helpers/token');
 
+const getUserObjectByToken = async token => {
+  const decodedToken = verifyToken(token);
+
+  if (decodedToken === 'error') {
+    return decodedToken;
+  }
+
+  const userObject = await obtainOneUser({ where: { email: decodedToken.email } });
+  return userObject;
```

Comment:
This var declaration is not necessary. You can do:

`return await obtainOneUser({ where: { email: decodedToken.email } });`